Gems/PhysX/Core/Code/Source/ArticulationLinkComponent.cpp

```diff
@@ -550,6 +550,10 @@ namespace PhysX
 
         PHYSX_SCENE_READ_LOCK(pxScene);
 
+        if (!m_articulation)
+        {
+            return;
+        }
         if (m_articulation->isSleeping())
         {
             return;
```

Comment:
```suggestion
        if (!m_articulation || m_articulation->isSleeping())
        {
            return;
```

It should be enough thanks to short-circuit evaluation.